internal/core/metadata/application/notify.go

```diff
@@ -198,3 +208,32 @@ func updateDeviceServiceCallback(ctx context.Context, dic *di.Container, ds mode
 		lc.Errorf("fail to invoke device service callback for updating device service %s, err: %s", ds.Name, response.Message)
 	}
 }
+
+// sendNotification sends a notification after adding or updating the device entity
+func sendNotification(ctx context.Context, dic *di.Container, deviceName string, httpMethod string) {
```

Comment:
rename `deviceName` to `name`, because we might send Notification for Profile, ProvisionWatcher, or others in the future.
rename `httpMethod` to `action`, because we should not be limited to REST API

Reply:
Modified.